### DISCUSSION TITLE
Confused and Looking to Buy Parts

### DISCUSSION THREAD

**Discussion by jamesondiskin** [sanni/cartreader#665] [Need help]
Hey there! I'm looking to build a V5 Cartridge Reader and am a little confused on what I need to purchase. Due to shipping times from AliExpress, I have already ordered some stuff that won't get delivered until around late February/early March. So far, the parts I have include the following:

- Replacement 180 degree N64 controller socket
- 50pcs Two Position 3 Pact Toggle Switches
- 2x MEGA 2560 Pro 16U

I plan to get the snesCIC for SA1 cartridges and understand that I need to get the DIP version. I also already own hundreds of dupont wires and resistors ready for the project, but I just wanted to make sure that everything I had in my AliExpress cart was good to go before placing the order. I know I still need the 100nF Ceramic Capacitor, but that is an eBay Listing per the Wiki's part list, so that is not included.

Does everything look okay here?
![image](https://user-images.githubusercontent.com/102488831/209199854-35c713e2-9d04-4053-ba98-610a114270af.png)
![image](https://user-images.githubusercontent.com/102488831/209199893-a9dfafd9-442c-4cb3-ab8b-0fe3c6a3922f.png)
![image](https://user-images.githubusercontent.com/102488831/209199925-36e07c8e-efa3-4d51-b8a0-138388d75a1c.png)
![image](https://user-images.githubusercontent.com/102488831/209199935-5af32480-3aaa-4541-aef6-8971c2dff7fe.png)

Sorry it doesn't fit all in one screenshot. Some of the quantities are higher because I am worried about messing up the soldering and I figured buying some replacement parts would be a smart choice.

Thanks for the help!

**Comment by sanni** [comment]
Looks good, you don't need the 1x8 20mm pin headers.
I have updated all the links in What-to-order wiki page and added a screenshot of the Aliexpress shopping cart for comparisson.

# Expected parts price for one unit     
[Picture of Aliexpress order](https://www.dropbox.com/s/63pptoy2fv9ahbl/aliexpress_order.png?dl=0) $94 incl. 19% Tax and Shipping      
[Picture of JLCPCB order](https://www.dropbox.com/s/dft06rmy16fq73s/jlcpcb_order.jpg?dl=0) $22 for PCBs, $17 for 3d printed frame = $40 incl. 19% Tax and Shipping      
**Total:** $134

Source: https://github.com/sanni/cartreader/wiki/What-to-order
